lib/__tests__/_sendEvent.test.ts

```diff
@@ -41,17 +43,21 @@ describe("sendEvent", () => {
     });
     it("should pass over the payload with multiple events", () => {
       (AlgoliaInsights as any).sendEvent("click", {
-        objectID: "1"
+        eventName: "my-event",
+        indexName: "my-index",
+        objectID: "1",
+        position: 3
       });
       expect(XMLHttpRequest.send).toHaveBeenCalledTimes(1);
       const payload = JSON.parse(XMLHttpRequest.send.mock.calls[0][0]);
       expect(payload).toEqual({
         events: [
           expect.objectContaining({
             eventType: "click",
-            objectID: "1",
-            userID: "42",
-            timestamp: expect.any(Number)
+            eventName: "my-event",
+            indexName: "my-index",
+            objectID: ["1"],
```

Comment:
Since the value is an array shouldn't be `objectIDs`?

Reply:
The endpoint accepts only `objectID: string[]`.
In the RFC, we agreed that we'll expose this as `objectIDs: string[]` because it gives a better hint on the nature of it.
```js
AlgoliaInsights.click({  objectIDs: ["1"], positions: [ 1 ] }) 
// which will call 
AlgoliaInsights._sendEvent({ eventType: "click", objectID: ["1"], position: [ 1 ] }) // signature is exactly matching payload requirement on the endpoint
// which will call 
XMLHttpRequest.send({ eventType: "click", objectID: ["1"], position: [ 1 ] })
```
If this is clear, please share your thoughts 🙂